test/e2e_env/universal/gateway/gateway.go

```diff
@@ -34,6 +34,7 @@ func Gateway() {
 	})
 
 	E2EAfterAll(func() {
+		Expect(env.Cluster.DeleteApp("gateway-client")).To(Succeed())
```

Comment:
This can/should really be fixed in a separate PR to `master` assuming it's unrelated

Reply:
https://github.com/kumahq/kuma/pull/5513/files